src/LanguageServer/Impl/Formatting/LineFormatter.cs

```diff
@@ -180,6 +180,7 @@ public class LineFormatter {
                         break;
 
                     case TokenKind.Assign:
+                    case TokenKind.ColonEqual:
```

Comment:
It's functionally the same, but given `:=` is an operator which appears in an expression like `+`, I would probably have put it with the other normal operators.

Reply:
Moved it

Reply:
You moved it with the augmented assignment operators; I meant to move it with "Operators" a few lines down.

Reply:
Fixed